terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/java/org/terasoluna/gfw/web/pagination/PaginationTag.java

```diff
@@ -266,57 +266,22 @@ protected void writePageLink(TagWriter tagWriter, PaginationInfo info,
     protected void writeFirstAndPreviousLink(TagWriter tagWriter,
```

Comment:
https://github.com/terasolunaorg/terasoluna-gfw/blob/2292379260a5834e2a230a413a679d466a6aa26d/terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/main/java/org/terasoluna/gfw/web/pagination/PaginationTag.java#L238-L239
Have you considered "writePageLink"?
You may be able to fix it in the same way.

Reply:
21f0779a3aa468ee4514be0de26e66dad68f641a
I modified "writePageLink" in the same way.